client/client.go

```diff
@@ -307,6 +315,14 @@ func (c *Config) Alertf(message string, format ...any) {
 	l.MoreData = json.RawMessage("{}")
 	c.dispatchLogMessage(l)
 
+	if c.AlertWebhook != nil {
+		err := c.AlertWebhook(l)
+		if err != nil {
+			// log error sending webhook inline to avoid infinite loop and ignoring the message
+			c.Slogger.Log(context.Background(), barkslogger.LvlAlert, "unable to send webhook")
```

Comment:
We have to check if the Slogger is nil or not.

Reply:
and if it is, then we need to `fmt.Println` that entry.

Reply:
Would you like a full models.Barklog printed (presumably as json) or just the string "unable to send alert"?

Reply:
The model struct can be used as a string. It formats itself. Look at this: https://github.com/techrail/bark/blob/3f5384a43eb7a41b587d5d7026865c981b20d4db/models/barklog.go#L63